homeworks/nikolay.krasniuk_Kingstar22/5-rxjs/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>5-RXJS</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/rxjs/6.6.3/rxjs.umd.min.js" integrity="sha512-JnZPWAckMDks0CCIeGznqEo7ifmYK7VowHq/NVFlSXGkwxv4OBCJFy7nouOtJZmVldHo7FQLIdeWTI55yyZc4Q==" crossorigin="anonymous"></script>
+    </head>
+    <body>
+        <section class="main">
+            <header data-header-menu class="main__header active">
+                <div class="main__wrapper">
+                    <a class="main__header-logo">Logo</a>
+                    <div data-header-burger class="main__header-burger">
+                        <span></span>
```

Comment:
Why do you need an empty span? If you don't, you can remove it

Reply:
i use span to create a burger menu

Reply:
Are you talking about assigning a tag class?

Reply:
Yes, it's better to add class. Thank you!